lace/archive/nyx_archive.py

```diff
@@ -44,9 +44,26 @@ def __init__(self, file_name=None, kp_Mpc=0.7, verbose=False, zmax=None):
         ## done set simulation list
 
         # list of available redshifts at Nyx (not all sims have them)
-        #self.list_sim_redshifts=np.append(np.arange(2.0,4.5,0.2),
+        # self.list_sim_redshifts=np.append(np.arange(2.0,4.5,0.2),
         #        np.arange(4.6,5.5,0.4))
-        self.list_sim_redshifts=[2,2.2,2.4,2.6,2.8,3,3.2,3.4,3.6,3.8,4,4.2,4.4,4.6,5,5.4]
+        self.list_sim_redshifts = [
+            2,
+            2.2,
+            2.4,
+            2.6,
+            2.8,
+            3,
+            3.2,
+            3.4,
+            3.6,
+            3.8,
+            4,
+            4.2,
+            4.4,
+            4.6,
+            5,
+            5.4,
+        ]
```

Comment:
Do we really need to use this format?

Reply:
My text editor only generates lines that are PEP compatible with a maximum length of 76? characters (I do not remember the exact value). I am not the one changing the format, this is automatic. We discussed this a long time ago, but I you prefer to use a non-PEP format in LaCE I am happy to disable this option from the text editor.

Reply:
I'm happy limiting the lines to < 80 characters (I usually do this because I work with vim), but I don't think we need to have 12 lines to define a range of redshifts... 

My main concern with the PEP formatting is that it hides changes in the code. I see all these diffs, and 95% of them are PEP-related changes, and the actual code changes are hidden. It would be great if we could have cleaner PR (or at least commits) with only code changes... lets talk abiut this next week with Laura.

Reply:
I guess there are three options:
- We all adhere to PEP guidelines
- We forget about this
- I first commit format changes, and then actual code changes